test/integration/functional_test.go

```diff
@@ -897,6 +898,31 @@ func validateDryRun(ctx context.Context, t *testing.T, profile string) {
 	}
 }
 
+func validateInternationalLanguage(ctx context.Context, t *testing.T, profile string) {
```

Comment:
add comment here so it shows up in the docs

Reply:
Done.